packages/resource-deployment/README.md

```diff
@@ -9,7 +9,19 @@ Before you start, make sure you have the Owner permission, or Contributor with U
 
 The deployment script is going to create Resource Group and deploy Storage Account, Batch Account and CosmosDB under it.
 
-### 1. Clone the repository
+### 1. Accept Azure Marketplace legal terms
+
+Before to deploy the Azure Batch pool virtual machine image you need to accept the Azure Marketplace legal terms one time on Azure subscription level. To accept legal terms use PowerShell Get-AzureRmMarketplaceTerms and Set-AzureRmMarketplaceTerms [API](https://go.microsoft.com/fwlink/?linkid=862451).
```

Comment:
"Before deploying" or "Before using the azure batch pool image"

Reply:
Updated